lib/trello_wrapper.rb

```diff
@@ -82,11 +75,24 @@ def set_description(card_id, description)
 
   private
 
+  def client
```

Comment:
Thanks for the feedback @jimmykarily, and sorry about the long delay. This is a rather old branch, so I have rebased it against current master and memoized the Trello client as you suggested. :slightly_smiling_face: 

Not sure if it's suitable to be merged after this long. Hopefully it still is.